core-bundle/src/Image/ImageFactoryInterface.php

```diff
@@ -22,9 +22,9 @@ interface ImageFactoryInterface
     /**
      * Creates an Image object.
      *
-     * @param string|ImageInterface              $path    The absolute path to the source image or an Image object
-     * @param int|array|ResizeConfiguration|null $size    An image size ID, an array with width, height and resize mode or a ResizeConfiguration object
-     * @param string|ResizeOptions|null          $options The target path as string or a ResizeOptions object
+     * @param string|ImageInterface                     $path    The absolute path to the source image or an Image object
+     * @param string|int|array|ResizeConfiguration|null $size    An image size ID, an array with width, height and resize mode or a ResizeConfiguration object
```

Comment:
```suggestion
     * @param ResizeConfiguration|array|int|string|null $size    An image size ID, an array with width, height and resize mode or a ResizeConfiguration object
```

Let's use the same order as in the actual signature.

Reply:
fixed (all)